pkg/docker-engine/Dockerfile

```diff
@@ -140,8 +140,8 @@ ENV GOPATH="/go"
 ENV GOTOOLCHAIN="local"
 ENV GO111MODULE="off"
 ENV PATH="$PATH:/usr/local/go/bin:$GOPATH/bin"
-ENV DOCKER_BUILDTAGS seccomp selinux
-ENV RUNC_BUILDTAGS seccomp selinux
+ENV DOCKER_BUILDTAGS="seccomp selinux"
+ENV RUNC_BUILDTAGS="seccomp selinux"
```

Comment:
These build tags may be obsolete; at least for runc, the default is not needed (but when set to empty, it will disable `seccomp`, so we should either remove the `RUNC_BUILDTAGS`, or at most keep just `seccomp`; https://github.com/opencontainers/runc/?tab=readme-ov-file#build-tags

- https://github.com/docker/containerd-packaging/pull/284

Reply:
And for docker;

- https://github.com/docker/docker-ce-packaging/pull/693